src/test/java/org/takes/facets/auth/PsByFlagTest.java

```diff
@@ -107,9 +111,15 @@ public void exitTest() throws IOException {
      * @throws Exception If some problem inside
      */
     @Test
-    public void equalsAndHashCodeEqualTest() throws Exception {
-        EqualsVerifier.forClass(PsByFlag.class)
-            .suppress(Warning.TRANSIENT_FIELDS)
-            .verify();
+    public void mustEvaluateTrueEqualityTest() throws Exception {
+        final Map<Pattern, Pass> passes = new HashMap<>(1);
+        passes.put(Pattern.compile("key"), new PsFake(true));
+        new Assertion<>(
+            "Must evaluate true equality",
+            new PsByFlag(passes),
+            new IsEqual<>(
```

Comment:
@fabriciofx put these 3 lines in the one line.

Reply:
@fabriciofx I don't think it looks better if I change.
![image](https://user-images.githubusercontent.com/3167940/69074618-ba962780-09fd-11ea-9d6d-f4de713f6410.png)